.github/workflows/docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Markdown Linter
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**/*.md'
+  pull_request:
+    paths-ignore:
+      - './**/*.md'
+      - './.vscode/**'
+      - './.husky/**'
+jobs:
+  test:
+    permissions:
+      contents: read
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+      - name: Install tools from asdf config
+        uses: ai/asdf-cache-action@v1
+      - name: Install dependencies
+        run: pnpm install --ignore-scripts
+      - name: Run tests
+        run: pnpm test:markdown
+        env:
+          FORCE_COLOR: 1
```

Comment:
We do not need `FORCE_COLOR` here. In `pnpm test` we need it because `pnpm` has an issue with nested tasks.